src/runtime/CMakeLists.txt

```diff
@@ -217,17 +217,31 @@ foreach (i IN LISTS RUNTIME_CPP)
                     # unfortunately, clang doesn't automatically set this flag even though the
                     # ABI is msvc on windows
                     set(fshort-wchar -fshort-wchar)
-                    set(TARGET "le64-unknown-windows-unknown")
+                    if (LLVM_PACKAGE_VERSION VERSION_LESS 19.0)
+                        set(TARGET "le64-unknown-windows-unknown")
+                    else ()
+                        # TODO: was le64 here, not sure if this is correct or not
+                        set(TARGET "aarch64-unknown-windows-unknown")
+                    endif ()
```

Comment:
What prevents us from just using `aarch64`/`arm` on all LLVM versions? I'm a bit worried about bugs in Halide 19 that appear only with LLVM 19 and not LLVM 18.

Also, does this introduce a requirement that LLVM be built with `ARM` and `AArch64` backends? We will need to check that, if so. #2282 will rear its head again.

Reply:
> What prevents us from just using aarch64/arm on all LLVM versions?

Nothing AFAICT. I'll make the change.

> Also, does this introduce a requirement that LLVM be built with ARM and AArch64 backends

Yes.